eng/targets/Settings.props

```diff
@@ -11,7 +11,7 @@
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
 
     <CommonExtensionInstallationRoot>CommonExtensions</CommonExtensionInstallationRoot>
-    <LanguageServicesExtensionInstallationFolder>Microsoft\ManagedLanguages\VBCSharp\LanguageServices</LanguageServicesExtensionInstallationFolder>
+    <LanguageServicesExtensionInstallationFolder>Microsoft\VBCSharp\LanguageServices</LanguageServicesExtensionInstallationFolder>
```

Comment:
i'm curious why we don't just reduce this to `Microsoft\VBCSharp`.  Is the LanguageServices part helpful?

Reply:
Both `LanguageServices` and `ExpressionEvaluator` are extensions that have been extracted, so they have a few overlapping files, like `manifest.json` etc.

I did propose on an email thread shortening them to `LS` and `EE`, or we could do `VBCSLanguageServices` and `VBCSExpressionEvaluator` directly under the `Microsoft` folder. To be perfectly frank the "Microsoft" bit seems a little redundant even :)